core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -93,12 +93,14 @@ public static void initializeNatives(Path tmpFile, boolean mlockAll, boolean sec
                 throw new RuntimeException("don't run elasticsearch as root.");
             }
         }
-        
+
         // enable secure computing mode
         if (seccomp) {
             Natives.trySeccomp(tmpFile);
+        } else {
+            logger.warn("Not enabling seccomp because it was explicitly disabled");
         }
```

Comment:
I agree that the dependency resolution is funky here. I think we'd be better off with dependency resolution handled by maven/gradle/whatever then how it works now. I suspect the reason the resolution isn't done that way now is because these are left over from single module days.

I think giving the test process some degree of control of the bwc nodes is useful though - it wants to be able to shut them down and spin up new nodes to replace them to simulate rolling upgrades which seems pretty fair.

The way I see this we need these tests working as soon as we want to make 2.0.1 so I don't know that we want to do major surgery to them. I'm all for figuring a way to get them working under seccomp and I'll work on it. With the goal of eventually moving them into the qa project with real resolution. And no elasticsearch nodes running in process with the tests.